src/cmd-cloud-prune

```diff
@@ -22,10 +22,11 @@
 #             "arches": [
 #                 "x86_64"
 #             ],
-#             "policy-cleanup": [
-#                 "cloud-uploads",
+#             "policy-cleanup": {
+#                 "cloud-uploads": true,
+#                 "images": true,
 #                 "images-kept": ["qemu", "live-iso"]
-#             ]
+#             }
```

Comment:
is changing this from a list going to have impact on any GC runs you've already done to non-production streams?

Reply:
Yes it will. The plan is to merge this, but before we run it, I will manually update the `builds.json` for rawhide and branched to have the `"policy-cleanup"` as a dict.